src/javax/bluetooth/UUID.java

```diff
@@ -0,0 +1,32 @@
+ 
+/*
+	This file is part of FreeJ2ME.
+
+	FreeJ2ME is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	FreeJ2ME is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with FreeJ2ME.  If not, see http://www.gnu.org/licenses/
+*/
+package javax.bluetooth;
+
+public abstract class UUID 
+{
+    public UUID(long uuidValue) { System.out.println("UUID Value:" + uuidValue); }
+
+    public UUID(String uuidValue, boolean shortUUID) { System.out.println("UUID Value:" + uuidValue); }
+
+    public abstract boolean equals(Object value);
+
+    public abstract int hashCode();
+
+    public abstract String toString();
+}
+ 
```

Comment:
While it doesn't really matter in Java, there's no reason not to add a newline at the end.